### PR TITLE
Add Cronos docs

### PR DESCRIPTION
**Pull request by ginstagram**
* Add quickstart, platform, operations, tutorials, blockchains sections for Cronos
* Add relnotes for 3-5-1, 3-5-5, 3-5-6 and Solana API docs
* Fix several typos

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -8,6 +8,43 @@ meta:
...
 # Release notes
 
+## Chainstack 3.5.6
+
+*October 3, 2022*
+
+### What's new
+
+* **Protocols**. Ethereum [Sepolia testnet support](/operations/ethereum/networks) for full nodes.
+* **Documentation**. Solana API [reference](/api/solana).
```
Comment (akegaviar) [inline docs/release-notes.md:18]: the correct link is `/api/solana/solana-api-reference`

File: docs/tutorials/cronos/nft-dutch-auction-contract-with-hardhat.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+## Overview
+
+To get from zero to deploying your own Dutch auction smart contract on the Cronos testnet, do the following:
+
+1. With Chainstack, create a [public chain project](https://docs.chainstack.com/glossary/public-chain-project).  
+1. With Chainstack, join the Cronos testnet.
+1. With Chainstack, access your Cronos node endpoint.
+1. With Hardhat, set up your development environment.
+1. With Hardhat, create and compile your Dutch auction contract.
+1. With Metamask, fund your wallet with test CRO tokens from the [Cronos official faucet](https://cronos.org/faucet).
```
Comment (akegaviar) [inline docs/tutorials/cronos/nft-dutch-auction-contract-with-hardhat.md:36]: MetaMask

File: docs/tutorials/cronos/nft-dutch-auction-contract-with-hardhat.md [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+
+Specifically, in this tutorial, you will:
+
+* Create a Dutch auction smart contract.
+* Deploy and verify the contract on the Cronos testnet through a node deployed with Chainstack.
+* Interact with the deployed contract.
+
+## Prerequisites
+
+* <a href="https://console.chainstack.com/" target="_blank">Chainstack account</a> to deploy a Cronos Testnet node.
```
Comment (akegaviar) [inline docs/tutorials/cronos/nft-dutch-auction-contract-with-hardhat.md:23]: `testnet`

File: docs/tutorials/cronos/nft-dutch-auction-contract-with-hardhat.md [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+
+See [View node access and credentials](/platform/view-node-access-and-credentials).
+
+### Install Hardhat
+
+See [Hardhat documentation](https://hardhat.org/hardhat-runner/docs/getting-started#installation).
+
+### Initialize a Hardhat project
+
+In your project directory, run `npx hardhat`. Select Create a JavaScript project and agree to install the sample project's dependencies. This will create a sample project directory with a smart contract draft.
```
Comment (akegaviar) [inline docs/tutorials/cronos/nft-dutch-auction-contract-with-hardhat.md:60]: `Select Create a JavaScript `?

If it's a UI element, it needs proper style. For example, in bold.

File: docs/tutorials/cronos/nft-dutch-auction-contract-with-hardhat.md [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+
+2. To compile the contract, in your project directory, run:
+
+```js
+npx hardhat compile
+```
+
+### Fund your account
+
+To deploy your smart contract on the Cronos testnet, you will need some Test CRO tokens. See the [Cronos faucet](https://cronos.org/faucet).
```
Comment (akegaviar) [inline docs/tutorials/cronos/nft-dutch-auction-contract-with-hardhat.md:165]: Is it `Test CRO` or `test CRO`? There are different versions here in the tutoril.

File: docs/release-notes.md [hunk 6]
```diff
@@ -8,6 +8,43 @@ meta:
...
 # Release notes
 
+## Chainstack 3.5.6
+
+*October 3, 2022*
+
+### What's new
+
+* **Protocols**. Ethereum [Sepolia testnet support](/operations/ethereum/networks) for full nodes.
+* **Documentation**. Solana API [reference](/api/solana-api-reference).
```
Comment (akegaviar) [inline docs/release-notes.md:18]: `/api/solana/solana-api-reference`

**Comment by akegaviar** [comment]
Change the protocol order everywhere in the commit please to:

* Cronos
* Fantom

**Review by akegaviar — CHANGES_REQUESTED** [review]
Some minor changes

**Review by akegaviar — APPROVED** [review]

**Review by NikitosnikN — APPROVED** [review]

**Pull request merged**